src/vs/workbench/browser/parts/editor/media/multieditortabscontrol.css

```diff
@@ -293,13 +293,17 @@
 	background-color: var(--tab-border-top-color);
 }
 
-.monaco-workbench .part.editor > .content .editor-group-container > .title .tabs-container > .tab.active.tab-border-bottom:not(:focus) > .tab-border-bottom-container {
+.monaco-workbench .part.editor > .content .editor-group-container > .title .tabs-container > .tab.active.tab-border-bottom > .tab-border-bottom-container {
 	z-index: 10;
 	bottom: 0;
 	height: 1px;
 	background-color: var(--tab-border-bottom-color);
 }
 
+.monaco-workbench .part.editor > .content .editor-group-container > .title .tabs-container > .tab.active.tab-border-bottom:focus > .tab-border-bottom-container {
+	background-color: var(--vscode-focusBorder);
```

Comment:
borders are rendered above outlines. Due to this the bottom border of the tab will render above the bottom border of the outline. In this case we switch the bottom border to have the focus outline color creating the illusion of it being a focus outline.